16x16.js/sketch.js

```diff
@@ -66,11 +67,11 @@ function keyPressed(e) {
 
 
 const renderGrid = (x = 0, y = 0, width = 400, height = 400) => {
-  grid.update()
-
-  const fontSize = u * 0.75
-
+  grid.renderSequence()
```

Comment:
Refactored `grid.update()` to ` grid.renderSequence()` to better refelct what is happening.